rev_news/drafts/edition-116.md

```diff
@@ -40,9 +40,8 @@ __Various__
 
 __Light reading__
 
-<!---
-__Easy watching__
--->
```

Comment:
Please don't remove such commented sections. They are commented because we might want to uncomment them if we have relevant content (here videos) to put in them, and it's still possible at this point that we will have such content.

Reply:
Ah, apologies. Fixed. Was too hasty and accidentally deleted that section.